test/tests/make_install/tests

```diff
@@ -34,6 +34,7 @@
                  mkdir -p ../../../make_install_test'
       use_shell = True
       installation_type = IN_TREE
+      hpc = False
```

Comment:
yeah, this is removed now. `use_shell=True` doesn't work with HPC because we escape all shell commands when producing the qsub file. so now it's redundant here because `use_shell=True` will cause the skip